politeiad/backendv2/tstorebe/plugins/comments/cmds.go

```diff
@@ -1183,29 +1193,61 @@ func (p *commentsPlugin) cmdVotes(token []byte, payload string) (string, error)
 		return "", err
 	}
 
-	// Compile the comment vote digests for all votes that were cast
-	// by the specified user.
-	digests := make([][]byte, 0, 256)
```

Comment:
according to this line in the old implementation the reply was limited to 256 votes.
would it make sense to declare the default page size as 256 to ensure that this is
backward compatible ?

@lukebp

Reply:
That's not what that means.  That's a pre-allocation that is used for performance reasons.  Learn up on it.

Reply:
oh true! 
sorry misread it, it just the initial size